hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1480,45 +1535,36 @@ def test_not_equal(x, y):
             key=lambda i: st.buffer[examples[i].start : examples[i].end],
         )
 
-    def run_block_program(self, i, description, original, repeats=1):
-        """Block programs are a mini-DSL for block rewriting, defined as a sequence
-        of commands that can be run at some index into the blocks
+    def run_node_program(self, i, description, original, repeats=1):
+        """Node programs are a mini-DSL for node rewriting, defined as a sequence
+        of commands that can be run at some index into the nodes
 
         Commands are:
 
-            * "-", subtract one from this block.
-            * "X", delete this block
-
-        If a command does not apply (currently only because it's - on a zero
-        block) the block will be silently skipped over.
+            * "X", delete this node
```

Comment:
If I'm reading this correctly, we now handle the `-` case in `minimize_individual_nodes` instead?

Reply:
sort-of-not-really? `-` was only used for these weird `-XX` and `--X` passes, which were cleanly removable save for `test_pandas_hack`.  `-` only ever decremented a block by one while `minimize_individual_nodes ` has some stronger logic per node. The `-` command doesn't exist anymore, though it could still make semantic sense in the future, where instead of `n - 1` it means the next lowest entropy value for the node.

Reply:
Got it, thanks.  Also as above I was mixing up Block/Example too.